### ISSUE TITLE
Page - study at hack your future

### ISSUE THREAD

**Issue by KevinTss** [HackYourFutureBelgium/hackyourfuture.be#250]
# Context

This page should target students (Potential or alumni)

- Who are HYF Belgium students?
- Who can apply
- How apply
- What's the projection as student who finish HYF Belgium curriculum?

### ...

Work in progress

**Comment by sebherrerabe** [comment]
Is this the equivalent to the current "Our program" page ?

**Comment by KevinTss** [comment]
> Is this the equivalent to the current "Our program" page ?

Yes exactly ([here](https://hackyourfuturebelgium.be/program/)), but the rework version, we'll do a V1 then iterate on it indeed
